python/torch_mlir_e2e_test/test_suite/rng.py

```diff
@@ -52,9 +52,9 @@ def __init__(self):
     @export
     @annotate_args([
         None,
-        ([256, 512, 8], torch.float64, True),
-        ([512, 1024, 4], torch.float64, True),
-        ([512, 256, 4], torch.float64, True),
+        ([256, 512, 12], torch.float64, True),
```

Comment:
nit: Is there a particular reason why this changed?

Reply:
Oh, then can we just make the sizes dynamic if they aren't important?

(not a big deal either way)